### PR TITLE
Documents update

### PR DESCRIPTION
**Pull request by marissaDubbelaar**
## PR checklist
- [X] Documentation in `docs` is updated

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Contributions to the code are even more welcome ;)
 If you'd like to write some code for nf-core/epitopeprediction, the standard workflow is as follows:
 
 1. Check that there isn't already an issue about your idea in the [nf-core/epitopeprediction issues](https://github.com/nf-core/epitopeprediction/issues) to avoid duplicating work
-    * If there isn't one already, please create one so that others know you're working on this
+    * If there isn't one, please create one so that others know you're working on this
```
Comment (ggabernet) [inline .github/CONTRIBUTING.md:19]:
```suggestion
    * If there isn't one already, please create one so that others know you're working on this
```

Comment (ggabernet) [reply .github/CONTRIBUTING.md:19]: Editing this file is what causes the nf-core lint tests to fail, so it should be kept like in the nf-core template.

Comment (ggabernet) [inline .github/CONTRIBUTING.md:19]:
```suggestion
    * If there isn't one already, please create one so that others know you're working on this
```

Comment (ggabernet) [reply .github/CONTRIBUTING.md:19]: Editing this file is what causes the nf-core lint tests to fail, it should be kept like the template :)

File: docs/output.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 ## Introduction
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline. At the end, version of the tools are summarized in a MultiQC report which is generated at the end of the pipeline.
```
Comment (ggabernet) [inline docs/output.md:9]:
```suggestion
This document describes the output produced by the pipeline. The version of all tools used in the pipeline are summarized in a MultiQC report which is generated at the end of the pipeline.
```

**Review by ggabernet — APPROVED** [review]
Just a small comment to fix the tests, otherwise looks good to me!

**Review by jonasscheid — APPROVED** [review]
Looks good to me!

**Pull request merged**